app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,9 @@
     <string name="main_activity_text_view_title">EstacionAR</string>
     <string name="main_activity_button_set_parking">Establecer estacionamientos</string>
     <string name="main_activity_toast_set_parking_confirmed">Se establecieron %1$d estacionamientos.</string>
+    <string name="dialog_fragment_title">Establecer estacionamientos</string>
+    <string name="dialog_fragment_button_ok">OK</string>
+    <string name="dialog_fragment_button_cancel">Cancelar</string>
+    <string name="dialog_fragment_textview_hint">Ingrese cantidad de espacios…</string>
+    <string name="dialog_fragment_toast_empty_input">Por favor ingrese un numero.</string>
```

Comment:
what will happen if you have another dialog fragment? please use more descriptive names

Reply:
strings names changed